tools/libSwiftScan/libSwiftScan.exports

```diff
@@ -0,0 +1,51 @@
+swiftscan_dependency_result_get_main_module_name
```

Comment:
You may have some entertaining conflicts with [this](https://man7.org/linux/man-pages/man5/exports.5.html).